tests/robot-tests/tests/general_public/prod_table_tool.robot

```diff
@@ -0,0 +1,131 @@
+*** Settings ***
+Resource            ../libs/public-common.robot
+
+Suite Setup         user opens the browser
+Suite Teardown      user closes the browser
+
+Force Tags          GeneralPublic    Prod
+
+*** Test Cases ***
+Go to 'data-tables' page
+    user navigates to data tables page on public frontend
+
+Check page contains correct themes
```

Comment:
Before this test case, I'd add a test case to count the number of themes using `user checks element count is x`. May need a little bit of care to make sure you're only selecting the details elements that are publications, rather than count all the themes and topics as well.

The best I can come up with is `xpath://fieldset[@id="publicationForm-publicationId"]/div/div/details`. It might be possible to get something better by adding a data-testid in the frontend code.

Reply:
I'd also check that the number of publications is correct - which could be done by checking the number of radio buttons on the page.